quarkchain/cluster/master.py

```diff
@@ -1828,6 +1840,22 @@ def parse_args():
     return env
 
 
+def start_grpc_server(env, master_server):
+    grpc_server = grpc.server(futures.ThreadPoolExecutor(max_workers=None))
+    servicer = ClusterMaster(master_server.root_state)
+    grpc_pb2_grpc.add_ClusterMasterServicer_to_server(servicer, grpc_server)
+    grpc_server.add_insecure_port(
+        "{}:{}".format(
+            env.cluster_config.GRPC_SERVER_HOST,
+            str(env.cluster_config.GRPC_SERVER_PORT),
+        )
+    )
+    try:
+        grpc_server.start()
+    except KeyboardInterrupt:
```

Comment:
since `grpc_server.start()` won't block here, I don't think catching `KeyboardInterrupt` is meaningful. should consider adding `grpc_server.stop` to the finished callbacks like jrpc